### PR TITLE
Scalefocus/xe module

### PR DESCRIPTION
**Pull request by DanielGL95**
Adding dependencies for xe exchange module and ldap module

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -355,6 +356,7 @@
         <!--spring.version>5.1.0.BUILD-SNAPSHOT</spring.version-->
         <spring-mobile.version>2.0.0.M2</spring-mobile.version>
         <spring-webflow.version>2.5.0.RELEASE</spring-webflow.version>
+        <spring-security-ldap.version>5.3.0.RELEASE</spring-security-ldap.version>
```
Comment (ptahchiev) [inline pom.xml:359]: This version is managed by the spring-security bom. You don't need to define it.

File: pom.xml [hunk 2]
```diff
@@ -178,6 +178,7 @@
         <graphql-spqr.version>0.0.4</graphql-spqr.version>
         <gridgain.version>6.6.9</gridgain.version>
         <guava.version>28.1-jre</guava.version>
+        <reactor.version>0.1.0</reactor.version>
```
Comment (ptahchiev) [inline pom.xml:181]: Please rename to `xe-rates-client.version` and also try to order the properties alphabetically.

File: pom.xml [hunk 3]
```diff
@@ -701,6 +703,12 @@
                 <version>${spring-webflow.version}</version>
             </dependency>
 
+
+            <dependency>
```
Comment (ptahchiev) [inline pom.xml:707]: This dependency is managed by the spring security bom. You don't need to define it.

File: pom.xml (hunk 1 above)
Comment (DanielGL95) [reply pom.xml]: Fixed

File: pom.xml (hunk 2 above)
Comment (DanielGL95) [reply pom.xml]: Fixed

File: pom.xml (hunk 3 above)
Comment (DanielGL95) [reply pom.xml]: Fixed

File: pom.xml [hunk 4]
```diff
@@ -178,6 +178,7 @@
         <graphql-spqr.version>0.0.4</graphql-spqr.version>
         <gridgain.version>6.6.9</gridgain.version>
         <guava.version>28.1-jre</guava.version>
+        <xe-rates-client.version.version>0.1.0</xe-rates-client.version.version>
```
Comment (ptahchiev) [inline pom.xml:181]: why is it called xe-rates-client.version.version????

Comment (ptahchiev) [reply pom.xml:181]: Why is it not ordered alphabetically???

Comment (DanielGL95) [reply pom.xml]: Fixed

**Pull request merged**